packages/utils/animation.ts

```diff
@@ -3,4 +3,4 @@ export const cubic = (value: number): number => {
 }
 
 export const easeInOutCubic = (value: number): number =>
-  value < 0.5 ? cubic(value * 2) / 2 : 1 - cubic((1 - value) * 2) / 2
+  value < 0.5 ? cubic(value << 1) >> 1 : 1 - (cubic((1 - value) << 1) >> 1)
```

Comment:
It seems that it is not as easy to understand as before

Reply:
yes，But now it can have higher performance, and bit operation should be the basis. As a public module, it should have better performance?

Reply:
For readability, consider adding comments?

Reply:
If the `value` is a decimal (eg, 0.5), the calculation result is different from before.

Reply:
yes,I was careless， you`re right

Reply:
If you can not fix this, you can close this PR by yourself.